src/t2iapi/context/service.proto

```diff
@@ -105,6 +105,25 @@ service ContextService {
   t2iapi.context.CreateContextStateWithAssocAndSpecificValidatorRequest)
       returns (t2iapi.context.CreateContextStateWithAssociationResponse);
 
+  /*
+  Create or adapt a context state, such that
+  - it belongs to the given descriptor_handle,
+  - it has the given context_association value,
+  - it has the given number of pm:Validators (num_validators)
+  - its pm:Identification changes (in case the context state is newly created, it
+    must differ from the pm:Identification of the previously associated context state
```

Comment:
Since the manipulation could also be used for a pm:AbstractContextDescriptor that owns mutliple associated states, the changed identification should be different to all other identifications of associated states. What do you think?
Resp. different to everyone's identification (previously and currently associated states)?
Or we should limit the manipulation to pm:LocationContextDescriptors resp. pm:PatientContextDescriptors?

Reply:
Yes, indeed, thanks for pointing this out!

If I understood Lukas correctly, then the property we want to enforce is a one-to-one mapping between associated Location/PatientContextStates and associated EnsembleContextStates related to an EnsembleContextDescriptor. This does neither imply that the pm:Identifications of all EnsembleContextStates (associated or not) need to be unique not that the pm:Identification of all (associated or not) Location/PatientContextStates need to be unique.
I hence think that requiring this manipulation to guarantee overall uniqueness is besides the point.

However, requiring the newly created contextState to have a pm:Identification that differs from the ones of all associated ContextStates of the same kind makes sense, yes.
I will adapt the Documentation in this direction.

Reply:
Done.